tests/integration/helm_test.go

```diff
@@ -45,65 +41,42 @@ func TestHelmSuite(t *testing.T) {
 type HelmSuite struct {
 	suite.Suite
 	helper    *clients.TestClient
-	k8sh      *utils.K8sHelper
-	installer *installer.InstallHelper
-	hh        *utils.HelmHelper
+	kh        *utils.K8sHelper
 	o         contracts.TestOperator
 	namespace string
 }
 
 func (hs *HelmSuite) SetupSuite() {
 	hs.namespace = "helm-ns"
-	kh, err := utils.CreateK8sHelper(hs.T)
-	require.NoError(hs.T(), err)
-
-	hs.k8sh = kh
-	hs.hh = utils.NewHelmHelper()
-
-	hs.installer = installer.NewK8sRookhelper(kh.Clientset, hs.T)
-	hs.o = NewBaseTestOperations(hs.installer, hs.T, hs.namespace, true)
-
-	err = hs.installer.CreateK8sRookOperatorViaHelm(hs.namespace)
-	require.NoError(hs.T(), err)
-
-	require.True(hs.T(), kh.IsPodInExpectedState("rook-operator", hs.namespace, "Running"),
-		"Make sure rook-operator is in running state")
-
-	require.True(hs.T(), kh.IsPodInExpectedState("rook-agent", hs.namespace, "Running"),
-		"Make sure rook-agent is in running state")
-
-	time.Sleep(10 * time.Second)
-
-	err = hs.installer.CreateK8sRookCluster(hs.namespace, "bluestore")
-	require.NoError(hs.T(), err)
 
-	err = hs.installer.CreateK8sRookToolbox(hs.namespace)
-	require.NoError(hs.T(), err)
+	o := NewBaseTestOperations(hs.T, hs.namespace, "bluestore", "", true, false, 3)
```

Comment:
these 4-5 lines seem duplicated in a number of places. Could a helper function set this up?

Reply:
we use the help function set up for most test. This test is doing a multi cluster set up, so i had to create a new implementaton of  Setup interface.

Reply:
i think what would help here is multiple returns from the method. So it could look like this:
```
hs.op, hs.kh = NewBaseTestOperations(hs.T, hs.namespace, "bluestore", "", true, false, 3)
```

Reply:
and `SetUp()` always called immediately after `NewBaseTestOperations()`? If so, perhaps `NewBaseTestOperations()` should call `SetUp()` for you